subsystem/kubeapi_test.go

```diff
@@ -2274,6 +2274,23 @@ spec:
 			return err
 		}, "1m", "10s").Should(HaveOccurred())
 	})
+
+	It("deploy AgentClusterInstall with missing ClusterDeployment", func() {
```

Comment:
suggesting to add another step that will add cluster deployment to "fix" the issue and check that sync state is consistently true.

Reply:
done